### DISCUSSION TITLE
请教个问题，你的get_md方法使用requests.get无法获取具体的博文页面，最后返回的是js代码，无法解析，求解决方法

### DISCUSSION THREAD

**Discussion by Smile-Cats** [ds19991999/csdn-spider#4] [General]

**Comment by Smile-Cats** [comment]
`response=requests.get('https://blog.csdn.net/amusi1994/article/details/80545854',headers={"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.70 Safari/537.36"})
`response.text
返回是这样的：
'<html><script>\nvar arg1=\'7AF5707D454D70667AFAC07D0083B51ABAD57D4F\';\nvar _0x4818=[\'\\x63\\x73\\x4b\\x48\\x77\\x71\\x4d\\x49\',\'\\x5a\\x73\\x4b\\x4a\\x77\\x72\\x38\\x56\\x65\\x41\\x73\\x79\',\'\\x55\\x63\\x4b\\x69\\x4e\\x38\\x4f\\x2f\\x77\\x70\\x6c\\x77\\x4d\\x41\\x3d\\x3d\',\'\\x4a\\x52\\x38\\x43\\x54\\x67\\x3d\\x3d\',\'\\x59\\x73\\x4f\\x6e\\x62\\x53\\x45\\x51\\x77\\x37\\x6f\\x7a\\x77\\x71\\x5a\\x4b\\x65\\x73\\x4b\\x55\\x77\\x37\\x6b\\x77\\x58\\x38\\x4f\\x52\\x49\\x51\\x3d\\x3d\',\'\\x77\\x37\\x6f\\x56\\x53\\x38\\x4f\\x53\\x77\\x6f\\x50\\x43\\x6c\\x33\\x6a\\x43\\x68\\x4d\\x4b\\x68\\x77\\x

这应该不是编码问题，求指教

**Comment by mesondzh** [comment]
这个应该是csdn的反爬机制，脚本有时候正常，有时候不行，试试这个模拟浏览器的爬虫 [#commit6](https://github.com/ds19991999/csdn-spider/tree/f8a47d20c13ad2af034b6a54ba1682f77187c36d)

**Comment by Smile-Cats** [comment]
老哥，不管是browser.get,还是session.get都没办法，

**Comment by mesondzh** [comment]
不会吧，为啥我可以用。。。

**Comment by Smile-Cats** [comment]
已经解决了，是因为selenium我没有设置等待时间，谢啦，

**Comment by mesondzh** [comment]
详见：https://github.com/ds19991999/csdn-spider/issues/3
